src/main/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockito.java

```diff
@@ -0,0 +1,537 @@
+
+                        Map<J.MethodDeclaration, JavaType.Method> mockedMethods = getMockUpMethods(newClass);
+
+                        // Add mockStatic field
+                        if (mockedMethods.values().stream().anyMatch(m -> m.getFlags().contains(Static))) {
+                            cd[0] = JavaTemplate.builder("private MockedStatic #{};")
+                                    .contextSensitive()
+                                    .javaParser(JavaParser.fromJavaVersion().classpathFromResources(ctx, MOCKITO_CLASSPATH))
+                                    .imports(MOCKITO_STATIC_IMPORT)
+                                    .staticImports(MOCKITO_ALL_IMPORT)
+                                    .build()
+                                    .apply(
+                                            new Cursor(getCursor().getParentOrThrow(), cd[0]),
+                                            cd[0].getBody().getCoordinates().firstStatement(),
+                                            MOCKITO_STATIC_PREFIX + className
+                                    );
+                            J.VariableDeclarations mockField = (J.VariableDeclarations) cd[0].getBody().getStatements().get(0);
+                            J.Identifier mockFieldId = mockField.getVariables().get(0).getName();
+                            tearDownMocks.put(MOCKITO_STATIC_PREFIX + className, mockFieldId);
+                        }
+                        // Add mockConstruction field
+                        if (mockedMethods.values().stream().anyMatch(m -> !m.getFlags().contains(Static))) {
+                            cd[0] = JavaTemplate.builder("private MockedConstruction #{};")
+                                    .contextSensitive()
+                                    .javaParser(JavaParser.fromJavaVersion().classpathFromResources(ctx, MOCKITO_CLASSPATH))
+                                    .imports(MOCKITO_CONSTRUCTION_IMPORT)
+                                    .staticImports(MOCKITO_ALL_IMPORT)
+                                    .build()
+                                    .apply(
+                                            updateCursor(cd[0]),
+                                            cd[0].getBody().getCoordinates().firstStatement(),
+                                            MOCKITO_CONSTRUCTION_PREFIX + className
+                                    );
+                            J.VariableDeclarations mockField = (J.VariableDeclarations) cd[0].getBody().getStatements().get(0);
+                            J.Identifier mockFieldId = mockField.getVariables().get(0).getName();
+                            tearDownMocks.put(MOCKITO_CONSTRUCTION_PREFIX + className, mockFieldId);
+                        }
+                    }));
+
+            cd[0] = maybeAddMethodWithAnnotation(this, cd[0], ctx, "tearDown",
+                    "@org.junit.After",
```

Comment:
Could you make all these constants - the After, org.junit.After so that there is more reuse? Also, could you handle junit 5 cases also ie AfterEach/BeforeEach and add a test case for this scenario? As otherwise the junit 5 + jmockit users will be excluded. Can do a separate PR for AfterAll/BeforeAll.